### PR TITLE
feat: integration toggle switch view

### PR DESCRIPTION
**Pull request by turrs**
integration component toggle view use context

### CODE REVIEW

File: src/Pages/Dashboard/index.jsx [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+  AllTrainingEvent,
+  SectionHeader,
+} from "../../Parts";
 const Dashboard = () => {
+  const { view } = useContext(AppContext);
   return (
     <div>
+      <SectionHeader />
       <FilterTrainingEvent />
+      {view ? <MyTrainingCard /> : <MyTrainingEventTable />}
```
Comment (argorfc) [inline src/Pages/Dashboard/index.jsx]: Please wrap in one element or use react fragment!

File: src/Pages/Dashboard/index.jsx [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+      {view ? <AllTrainingEvent /> : <AllTrainingEventTable />}
```
Comment (argorfc) [inline src/Pages/Dashboard/index.jsx]: Please wrap in one element or use react fragment!

File: src/Parts/MyTrainingCard/index.jsx [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+import { SingleTrainingCard } from "./SingleTrainingCard";
+import "./MyTrainingCard.css";
+import { useContext } from "react";
+import { AppContext } from "../../Context";
 
-export const MyTrainingCard = () => {
+const MyTrainingCard = () => {
   const { MyTrainingCardContext } = useContext(AppContext);
   return (
     <>
```
Comment (argorfc) [inline src/Parts/MyTrainingCard/index.jsx:11]: Change empty tag with react fragment

File: src/Parts/MyTrainingCard/index.jsx [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
       <div className="site-card-wrapper">
         <div className="title-event">
           <p>
-            My Training Event{' '}
+            My Training Event{" "}
```
Comment (argorfc) [inline src/Parts/MyTrainingCard/index.jsx]: Please use margin right to add space

File: src/Pages/Dashboard/index.jsx (hunk 1 above)
Comment (turrs) [reply src/Pages/Dashboard/index.jsx]: thankyou for review, this issue already fix in a7711523a75ebf1bee4167f348adc6eda02d0c44

File: src/Pages/Dashboard/index.jsx (hunk 2 above)
Comment (turrs) [reply src/Pages/Dashboard/index.jsx]: thankyou for review, this issue already fix in a7711523a75ebf1bee4167f348adc6eda02d0c44

File: src/Parts/MyTrainingCard/index.jsx (hunk 3 above)
Comment (turrs) [reply src/Parts/MyTrainingCard/index.jsx:11]: thankyou for review, this issue already fix in a7711523a75ebf1bee4167f348adc6eda02d0c44

File: src/Parts/MyTrainingCard/index.jsx (hunk 4 above)
Comment (turrs) [reply src/Parts/MyTrainingCard/index.jsx]: this issue will fix bye @aisyahnnd

**Review by argorfc — APPROVED** [review]
Approved.
Ready to merge.

**Pull request merged**